azure-pipelines/run_test_steps.yml

```
@@ -62,7 +64,18 @@ steps:
     env:
       TWISTED_REACTOR: ${{ parameters.windowsReactor }}
 
-- bash:  |
-    python -m tox -e coverage-prepare,coveralls-push
-    bash <(curl -s https://codecov.io/bash) -n "${{ parameters.platform }}-${{ parameters.pythonVersion }}-alldeps-withcov"
-  displayName: 'Report coverage'
+- bash: |
+    python -m coverage xml -o coverage.xml -i
+    bash <(curl -s https://codecov.io/bash) -X search -X gcov -f coverage.xml -v -n "${{ parameters.platform }}-${{ parameters.pythonVersion }}-alldeps-withcov"
+  displayName: 'Generate XMl and report codecov.io'
```

Capitalization and may as well match above I guess...

```suggestion
  displayName: 'Generate XML and publish to Codecov'
```